relnotes/AcquiredResources.migration.md

```diff
@@ -0,0 +1,5 @@
+### Module `swarm.neo.util.AcquiredResources` has been removed
+
+`swarm.neo.util.AcquiredResources`
+
+Please use `ocean.util.container.VoidBufferAsArrayOf` instead.
```

Comment:
I guess swarm internals had already been converted to use the ocean module?

Reply:
I'd think so. I found no existing usage of `swarm.neo.util.AcquiredResources` in swarm.